x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.tsx

```diff
@@ -63,6 +65,8 @@ export const TrustedAppsPage = memo(() => {
       }
       actions={addButton}
     >
+      <TrustedAppsNotifications />
```

Comment:
I mentioned the reasoning during our Office Hours meeting - it becomes cumbersome with selectors to implement it like that because the condition if it's open has to be checked twice - once in page component and once in dialog component. I suggest we move this discussion out and have a broader view on appropriate approach that fits the necessary characteristics and refactor this in one go.